tests/unittest/bindings/test_executor_bindings.py

```
@@ -66,6 +67,39 @@ def test_executor_from_memory(model_files, model_path):
+        "weight_int8":
+        np.array([[1, 2], [3, 4]], dtype=np.int8),
+        "weight_fp16":
+        np.array([[1.0, 2.0], [3.0, 4.0]], dtype=np.float16),
+        "weight_bf16":
+        torch_to_numpy(
+            torch.tensor([[1.0, 2.0], [3.0, 4.0]], dtype=torch.bfloat16)),
+        "weight_fp8":
+        torch_to_numpy(
+            torch.tensor([[1.0, 2.0], [3.0, 4.0]], dtype=torch.float8_e4m3fn)),
+    }
+
```

Ruff (0.12.2)

Local variable `executor` is assigned to but never used

Remove assignment to unused variable `executor`

(F841)